examples/14_standalone_h2_pem/standalone_h2.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<p>I'm not sure what you mean by "The <code>product_profile</code> will simply act as an "end cap"" -- could you maybe rephrase or clarify? Is it essentially a "sink" for an outputted commodity? I bet as I dig more into the code this will be more clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NREL/H2Integrate/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>